src/core/public/application/capabilities/capabilities_service.test.ts

```diff
@@ -61,6 +63,8 @@ Array [
       "body": "{\\"capabilities\\":{\\"navLinks\\":{\\"app2\\":true,\\"app1\\":true},\\"management\\":{},\\"catalogue\\":{},\\"app2\\":{\\"feature\\":true}}}",
       "credentials": "same-origin",
       "headers": Object {
+        "Content-Type": "application/json",
```

Comment:
optional: description says `calls backend API with merged capabilities`, but we test not only a request body, but also headers. That can lead to false positives when `http` service changes a header related setup

Reply:
This should probably be raised with security/capabilities.